### PR TITLE
alpine (2.21.99999)

### PR DESCRIPTION
**Pull request by ellio167**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
upgrade to 2.21.99999

### CODE REVIEW

File: Formula/alpine.rb [hunk 1]
```diff
@@ -1,19 +1,14 @@
 class Alpine < Formula
   desc "News and email agent"
   homepage "https://repo.or.cz/alpine.git"
-  url "https://ftp.osuosl.org/pub/blfs/conglomeration/alpine/alpine-2.21.tar.xz"
-  mirror "https://fossies.org/linux/misc/alpine-2.21.tar.xz"
-  sha256 "6030b6881b8168546756ab3a5e43628d8d564539b0476578e287775573a77438"
-  revision 2
-
-  bottle do
```
Comment (fxcoudert) [inline Formula/alpine.rb]: Do not remove or change the bottle block

File: Formula/alpine.rb [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-    sha256 "c3df47485dcedfed585bc0dbbb8fcbc2e6eed1494d48cf49a2ee224eba7e659e" => :catalina
-    sha256 "fa4a5f8078a26f5390325855dd98eae3ba60ca8c29cdb19ce6828e51862eef00" => :mojave
-    sha256 "96c62ab0c3a3f297f015f67add86563a365d77d477a4244588ad2e92d6f95e63" => :high_sierra
-  end
+  url "https://repo.or.cz/alpine.git/snapshot/v2.21.99999.tar.gz"
```
Comment (fxcoudert) [inline Formula/alpine.rb]: Where is this version announced as a released, stable version?

File: Formula/alpine.rb [hunk 3]
```diff
@@ -1,19 +1,14 @@
...
+  sha256 "6b27da784b8c1dd5bc2553cb04e622ac4461d58409a178b7c83541cd167451d0"
+  head "https://repo.or.cz/alpine.git"
 
   depends_on "openssl@1.1"
 
+  patch :DATA
```
Comment (fxcoudert) [inline Formula/alpine.rb]: Where does this patch come from? What bug does it fix? Has it been submitted to upstream and accepted as valid?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/alpine.rb (hunk 1 above)
Comment (ellio167) [reply Formula/alpine.rb]: Sorry, I will fix this and avoid this mistake in the future.

**Comment by ellio167** [comment]
It turns out, I misunderstood the release scheme/process used by upstream.  The current stable version is still 2.21.  Thanks for your help.

**Pull request closed without merging**